src/mam4xx/mam4.hpp

```diff
@@ -10,6 +10,7 @@
 
 #include <haero/aero_process.hpp>
 #include <mam4xx/aero_config.hpp>
+#include <mam4xx/aero_model.hpp>
```

Comment:
We should probably make sure this header gets installed when we type `make install` (in `src/mam4xx/CMakeLists.txt`).